README.md

```diff
@@ -1,2 +1,24 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+A) As the algorithm starts it requires importing list called "list". Then sets n as a length of a "list". Next 2 for loops go through every single object in "list and chectk whether there is the same value, if there is one duplicate script returns true and stops whole function but if there is not duplicates, algorithm will return false.
+
+B) First for loop is going to happen n times, and 2nd for loop inside is going to happen n times for each value in a first for loop making this algorithm quadratic.
+
+C) Algorithm was modified fo it takes half of time because every object in the list is connecting only once with the other ones rather than twice how it was before script was eddited.
+
+D) Algorithm will run aproximately twice as fast because there will be half less calculations because how algorithm is structured now.
+
+E) I think that algorithm will still be a quadratic buth with less steep distribution.
+
+F) Best case performance of this function is O(n) and the worst case performance is O(nlogn). Source: https://mail.python.org/pipermail/python-dev/2002-July/026837.html
+
+G) sort function = O(nlogn), linear function (for loop) = O(n) therefore this algorithm = O(nlogn + n) function will be 
```

Comment:
✔️ -- however this can be further simplified to just O(n log n), since n log n grows faster than n.